### PR TITLE
chore: enable jsdoc/sort-tags

### PR DESCRIPTION
**Pull request by Shinigami92**

### CODE REVIEW

File: src/word.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class Word {
    * Returns an adjective of random or optionally specified length.
    * If specified length is unresolvable, returns random adjective.
    *
-   * @param optional length of word to return
+   * @param length length of word to return
```
Comment (ST-DDT) [inline src/word.ts:18]:
```suggestion
   * @param length The optional length of word to return.
```

File: src/word.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ export class Word {
    * Returns an adverb of random or optionally specified length.
    * If specified length is unresolvable, returns random adverb.
    *
-   * @param optional length of word to return
+   * @param length length of word to return
```
Comment (ST-DDT) [inline src/word.ts:40]:
```suggestion
   * @param length The optional length of word to return.
```

File: src/word.ts [hunk 3]
```diff
@@ -59,7 +59,7 @@ export class Word {
    * Returns a conjunction of random or optionally specified length.
    * If specified length is unresolvable, returns random conjunction.
    *
-   * @param optional length of word to return
+   * @param length length of word to return
```
Comment (ST-DDT) [inline src/word.ts:62]:
```suggestion
   * @param length The optional length of word to return.
```

File: src/word.ts [hunk 4]
```diff
@@ -81,7 +81,7 @@ export class Word {
    * Returns an interjection of random or optionally specified length.
    * If specified length is unresolvable, returns random interjection.
    *
-   * @param optional length of word to return
+   * @param length length of word to return
```
Comment (ST-DDT) [inline src/word.ts:84]:
```suggestion
   * @param length The optional length of word to return.
```

File: src/word.ts [hunk 5]
```diff
@@ -103,7 +103,7 @@ export class Word {
    * Returns a noun of random or optionally specified length.
    * If specified length is unresolvable, returns random noun.
    *
-   * @param optional length of word to return
+   * @param length length of word to return
```
Comment (ST-DDT) [inline src/word.ts:106]:
```suggestion
   * @param length The optional length of word to return.
```

File: src/word.ts [hunk 6]
```diff
@@ -125,7 +125,7 @@ export class Word {
    * Returns a preposition of random or optionally specified length.
    * If specified length is unresolvable, returns random preposition.
    *
-   * @param optional length of word to return
+   * @param length length of word to return
```
Comment (ST-DDT) [inline src/word.ts:128]:
```suggestion
   * @param length The optional length of word to return.
```

File: src/word.ts [hunk 7]
```diff
@@ -147,7 +147,7 @@ export class Word {
    * Returns a verb of random or optionally specified length.
    * If specified length is unresolvable, returns random verb.
    *
-   * @param optional length of word to return
+   * @param length length of word to return
```
Comment (ST-DDT) [inline src/word.ts:150]:
```suggestion
   * @param length The optional length of word to return.
```

**Review by ST-DDT — CHANGES_REQUESTED** [review]

**Review by ST-DDT — APPROVED** [review]

**Comment by ST-DDT** [comment]
FFR: We expect the following order of tags: https://github.com/gajus/eslint-plugin-jsdoc/blob/master/src/defaultTagOrder.js

**Review by prisis — APPROVED** [review]

**Review by prisis — APPROVED** [review]
I love the example comments

**Pull request merged**